server/service/src/sync/test/integration/remote/program_requisition.rs

```diff
@@ -21,7 +21,7 @@ impl SyncRecordTester for ProgramRequisitionTester {
         // STEP 1 - insert
         let period_schedule1 = PeriodScheduleRow {
             id: uuid(),
-            name: "Weekly".to_string(),
+            name: uuid(),
```

Comment:
This one I was a little surprised hadn't failed before now? It does a find_one_by_name in the translation, which once you've run the integration tests a few times, always finds the first one, rather than this newly created one